config/initializers/public_availabilty.rb

```diff
@@ -0,0 +1,2 @@
+Rails.configuration.public_prisons_with_slot_availability =
```

Comment:
I think this should be an environment variable as we want to have different lists per environment. This is how the staff equivalent is also done